ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -146,7 +146,8 @@ class TabBarItemView: UIControl {
     }
 
     private struct Constants {
-        static let unselectedColor: UIColor = Colors.textSecondary
+        // TODO: to be updated
+        // static let unselectedColor: UIColor =
```

Comment:
Did you mean to have this comment? I see you moved it down to a private lazy var so it would have access to the theme.

Reply:
Whoops!